docs/multiplayer.md

```diff
@@ -1,225 +1,208 @@
-    - 4.1)[Imposing landscaping limits](#41-imposing-landscaping-limits)
+  - 4.1)[Imposing landscaping limits](#41-imposing-landscaping-limits)
 - 5.0) [Some useful things](#50-some-useful-things)
 - 6.0) [Troubleshooting](#60-troubleshooting)
 
 
 ## 1.0) Starting a server
 
- - Make sure that you have your firewall of the computer as well as possible
-   routers or modems of the server configured such that:
-    - port 3979 is free for both UDP and TCP connections in- and outgoing
-    - port 3978 is free outbound for UDP in order to advertise with the master
-     server (if desired). Otherwise you'll have to tell players your IP.
-    - port 3977 if use of the admin interface is desired (see admin_network.txt)
- - Click "multiplayer" on the startup screen
- - Click "start server"
- - Type in a game name
- - Select the type of game ('LAN/Internet' or 'Internet (advertise)'. With the
-   last one other people are able to see you online. Else they need your IP and
-   port to join)
- - Click "start game", "load game" or "load scenario"
- - Start playing
-
+- Click on "Multiplayer" in the Start Menu.
+- Click on "Start Server".
+- Give your server a name.
+- Select the visibility of your server:
+  - "Public": your server will be publicly listed.
+  - "Invite Only": only players who have the invite code for your server can join.
+  - "Local": only players on your local network can join.
+- (optional) Set a password for your server.
+- Click "New Game", "Load Game", or "Play Scenario".
+- Start playing.
 
 ## 2.0) Connecting to a server
 
- - Click "multiplayer" on the startup screen
- - If you want to connect to any network game in your LAN click on 'LAN', then
-   on 'Find Server'
- - If you want to see which servers all online on the Internet, click on
-   'Internet' and 'Find Server'
- - If there were more than one server
-    - select one in the list below the buttons
-    - click on 'join game'
- - If you want to play and you have the ip or hostname of the game server you
-   want connect to.
-    - click add server
-    - type in the ip address or hostname
-    - if you want to add a port use :<port>
- - If you want to play and you have the invite code of the game server you
-   want connect to.
-    - click add server
-    - type in the invite code
- - Now you can select a company and press: "Join company", to help that company
-    - Or you can press "Spectate game", to spectate the game
-    - Or you can press "New company", and start your own company (if there are
-   slots free)
- - You see a progressbar how far you are with joining the server.
- - Happy playing
+- Click on "Multiplayer" in the Start Menu.
+- There are three ways to join a server:
+  - If you want to connect to a local server, click "Search LAN".
+  - If you want to connect to a public game, click "Search internet".
+  - If the server-owner shared an invite code with you:
+    - Click "Add Server".
+    - Fill in the invite code, which always starts with a +.
```

Comment:
It seems the text uses double quotes for something that people see literally on the screen, so shouldn't the + be within double quotes as well?